README.md

```diff
@@ -36,8 +36,14 @@ graph LR
 
 See this project's [fixtures](test/fixtures) for more examples.
 
+### Options
+
+- `simple`: A boolean option which determines whether the plugin will generate SVG, or simply wrap your Mermaid graph in an `<div class="mermaid">` element which [mermaidjs][mermaidjs] can understand.
```

Comment:
Be clearer about what `true` does and what `false` does. So:

- `simple`: when set to true, plugin will wrap mermaid graphs in an `<div class="mermaid">` element instead of generating an SVG and linking to it.

Reply:
Fixed.